src/components/typography/text/text.spec.js

```diff
@@ -290,11 +322,19 @@ describe('<Body>', () => {
       });
     });
   });
+  describe('with dataTest', () => {
+    beforeEach(() => {
+      wrapper = shallow(<Text.Body data-test="prop-test">{'Body'}</Text.Body>);
```

Comment:
It would be nice to also test the `isInline` case and ensure data-props work with that as well since that uses a different rendering tree. If you have some time, feel free to test that case too. Not super important though.